### ISSUE TITLE
[Bug]:  No Tab present

### ISSUE THREAD

**Issue by NullEqualsZero** [huchenlei/sd-webui-openpose-editor#73]
### Is there an existing issue for this?

- [X] I have searched the existing issues and checked the recent builds/commits of both this extension and the webui

### What happened?

I installed via the extensions tab from Automatic1111 the extension does install but I don't see a tab anywhere 

### Steps to reproduce the problem

1. Go to extensions tab
2. Install the application 
3. Restart A1111
4. No tab found
![image](https://github.com/user-attachments/assets/63272105-ce1f-4d68-a051-a5e6b32a9d29)

### What should have happened?

I suppose a tab should be pressentpresent

### Commit where the problem happens

webui: 82a973c04367123ae98bd9abdf80d9eda9b910e2
controlnet: [56cec5b2](https://github.com/Mikubill/sd-webui-controlnet/commit/56cec5b2958edf3b1807b7e7b2b1b5186dbd2f81)
openpose-editor:[0026f453](https://github.com/huchenlei/sd-webui-openpose-editor/commit/0026f453f9ac9b7dec64129d2f2b83b04030c645)

### What browsers do you use to access the UI ?

Chrome

### Command Line Arguments

```Shell
--xformers --update-all-extensions (and sometime --listen --enable-insecure-extension-access)
```

### Console logs

```Shell
venv "E:\stable-diffusion-webui\venv\Scripts\Python.exe"
Python 3.10.10 (tags/v3.10.10:aad5f6a, Feb  7 2023, 17:20:36) [MSC v.1929 64 bit (AMD64)]
Version: v1.10.1
Commit hash: 82a973c04367123ae98bd9abdf80d9eda9b910e2
Pulled changes for repository in 'E:\stable-diffusion-webui\extensions\a1111-sd-webui-tagcomplete':
Already up to date.

Pulled changes for repository in 'E:\stable-diffusion-webui\extensions\adetailer':
Already up to date.

Pulled changes for repository in 'E:\stable-diffusion-webui\extensions\model-keyword':
Already up to date.

Pulled changes for repository in 'E:\stable-diffusion-webui\extensions\model_preset_manager':
Already up to date.

Pulled changes for repository in 'E:\stable-diffusion-webui\extensions\sd-extension-system-info':
Already up to date.

Pulled changes for repository in 'E:\stable-diffusion-webui\extensions\sd-webui-3d-open-pose-editor':
Already up to date.

Pulled changes for repository in 'E:\stable-diffusion-webui\extensions\sd-webui-animatediff':
Already up to date.

Pulled changes for repository in 'E:\stable-diffusion-webui\extensions\sd-webui-aspect-ratio-helper':
Already up to date.

Pulled changes for repository in 'E:\stable-diffusion-webui\extensions\sd-webui-controlnet':
Already up to date.

Pulled changes for repository in 'E:\stable-diffusion-webui\extensions\sd-webui-infinite-image-browsing':
Already up to date.

Pulled changes for repository in 'E:\stable-diffusion-webui\extensions\sd-webui-openpose-editor':
Already up to date.

Pulled changes for repository in 'E:\stable-diffusion-webui\extensions\sd_delete_button':
Already up to date.

Pulled changes for repository in 'E:\stable-diffusion-webui\extensions\sd_extension-prompt_formatter':
Already up to date.

Pulled changes for repository in 'E:\stable-diffusion-webui\extensions\Stable-Diffusion-Webui-Civitai-Helper':
Already up to date.

Pulled changes for repository in 'E:\stable-diffusion-webui\extensions\stable-diffusion-webui-dataset-tag-editor':
Already up to date.

Launching Web UI with arguments: --xformers --update-all-extensions
CHv1.8.11: Get Custom Model Folder
[-] ADetailer initialized. version: 24.9.0, num models: 16
ControlNet preprocessor location: E:\stable-diffusion-webui\extensions\sd-webui-controlnet\annotator\downloads
2024-09-29 14:16:12,273 - ControlNet - INFO - ControlNet v1.1.455
Loading weights [2be6f9698c] from E:\stable-diffusion-webui\models\Stable-diffusion\SD15\aniverse_v40Pruned\aniverse_v40Pruned.safetensors
CHv1.8.11: Set Proxy:
Creating model from config: E:\stable-diffusion-webui\configs\v1-inference.yaml
E:\stable-diffusion-webui\venv\lib\site-packages\huggingface_hub\file_download.py:1150: FutureWarning: `resume_download` is deprecated and will be removed in version 1.0.0. Downloads always resume when possible. If you want to force a new download, use `force_download=True`.
  warnings.warn(
Applying attention optimization: xformers... done.
Model loaded in 2.0s (load weights from disk: 0.1s, create model: 0.6s, apply weights to model: 1.1s, calculate empty prompt: 0.1s).
2024-09-29 14:16:15,083 - ControlNet - INFO - ControlNet UI callback registered.
E:\stable-diffusion-webui\extensions\model_preset_manager\scripts\main.py:446: GradioDeprecationWarning: 'scale' value should be an integer. Using 0.1 will cause issues.
  with gr.Column(min_width=100, scale = 0.1):
E:\stable-diffusion-webui\extensions\model_preset_manager\scripts\main.py:463: GradioDeprecationWarning: The `style` method is deprecated. Please set these arguments in the constructor instead.
  model_generation_data = gr.Textbox(label = model_generation_data_label_text(), value = "", lines = 3, elem_id = "def_model_gen_data_textbox").style(show_copy_button=True)
E:\stable-diffusion-webui\extensions\model_preset_manager\scripts\main.py:466: GradioDeprecationWarning: The `style` method is deprecated. Please set these arguments in the constructor instead.
  triggerWords = gr.CheckboxGroup([], multiselect=True, label="Trigger Words", interactive = True).style(container=True, item_container=True)
E:\stable-diffusion-webui\extensions\model_preset_manager\scripts\main.py:466: GradioDeprecationWarning: The `item_container` parameter is deprecated.
  triggerWords = gr.CheckboxGroup([], multiselect=True, label="Trigger Words", interactive = True).style(container=True, item_container=True)
E:\stable-diffusion-webui\extensions\model_preset_manager\scripts\main.py:493: GradioDeprecationWarning: The `style` method is deprecated. Please set these arguments in the constructor instead.
  output_textbox = gr.Textbox(interactive=False, label="Output").style(show_copy_button=True)
E:\stable-diffusion-webui\modules\gradio_extensons.py:25: GradioDeprecationWarning: `height` is deprecated in `Interface()`, please use it within `launch()` instead.
  res = original_IOComponent_init(self, *args, **kwargs)
[tag-editor] Custom taggers loaded: ['aesthetic shadow', 'cafeai aesthetic classifier', 'Improved Aesthetic Predictor', 'wd aesthetic classifier']
Running on local URL:  http://127.0.0.1:7860

To create a public link, set `share=True` in `launch()`.
IIB Database file has been successfully backed up to the backup folder.
Startup time: 25.1s (prepare environment: 12.7s, import torch: 3.2s, import gradio: 0.8s, setup paths: 0.9s, initialize shared: 0.2s, other imports: 0.5s, list SD models: 0.2s, load scripts: 2.9s, create ui: 3.1s, gradio launch: 0.3s).
```

### Browser logs

Running Stable-Diffusion-Webui-Civitai-Helper on Gradio Version: 3.41.2
index.js?1727603987.65641:1 sd-webui-3d-open-pose-editor: onUiLoaded
index.js?a83964bb:85 i18next: languageChanged en
index.js?a83964bb:85 i18next: initialized {debug: true, initImmediate: true, ns: Array(1), defaultNS: 'common', fallbackLng: Array(1), …}
index.js?a83964bb:85 0.1.18  2023-04-15 16:20:24
index-9fefa6dc.js:186 Could not find "window.__TAURI_METADATA__". The "appWindow" value will reference the "main" window label.
Note that this is not an issue if running this frontend on a browser instead of a Tauri window.
(anonymous) @ index-9fefa6dc.js:186Understand this warning
index-9fefa6dc.js:192 {BASE_URL: '/infinite_image_browsing/fe-static', MODE: 'production', DEV: false, PROD: true, SSR: false}
index-9fefa6dc.js:208 undefined
index-9fefa6dc.js:196 Splitpanes: Could not resize panes correctly due to their constraints.
(anonymous) @ index-9fefa6dc.js:196
Promise.then
ze @ index-9fefa6dc.js:1
readjustSizes @ index-9fefa6dc.js:196
initialPanesSizing @ index-9fefa6dc.js:196
resetPaneSizes @ index-9fefa6dc.js:196
mounted @ index-9fefa6dc.js:196
(anonymous) @ index-9fefa6dc.js:1
ea @ index-9fefa6dc.js:1
kn @ index-9fefa6dc.js:1
e.__weh.e.__weh @ index-9fefa6dc.js:1
E0 @ index-9fefa6dc.js:1
ye @ index-9fefa6dc.js:1
mount @ index-9fefa6dc.js:1
e.mount @ index-9fefa6dc.js:1
(anonymous) @ index-9fefa6dc.js:218Understand this warning
index.js?a83964bb:102 @mediapipe/pose Module {Symbol(Symbol.toStringTag): 'Module'}
index.js?a83964bb:102 MyPose ƒ Pv(h){var w=this;h=h||{},this.h=new Ch({locateFile:h.locateFile,files:function(M){return[{url:"pose_solution_packed_assets_loader.js"},{simd:!1,url:"pose_solution_wasm_bin.js"},{simd:!0,url:"pose_sol…
index.js?a83964bb:3440 create editor
(anonymous) @ index.js?a83964bb:3440
zd @ index.js?a83964bb:40
Da @ index.js?a83964bb:40
(anonymous) @ index.js?a83964bb:40
T @ index.js?a83964bb:25
ee @ index.js?a83964bb:25Understand this warning
index-9fefa6dc.js:208 restoreFeGlobalSetting Proxy(Object) {defaultChangeIndchecked: true, defaultSeedChangeChecked: false, darkModeControl: 'auto', dontShowAgainNewImgOpts: false, defaultSortingMethod: 'created-time-desc', …}
index.js?a83964bb:3305 Infinity% loaded
index.js?a83964bb:3419 method GetAppVersion []
index.js?a83964bb:3419 return {target: Window, origin: 'http://127.0.0.1:7860', data: {…}}
index.js?1727603987.65641:1 Method GetAppVersion MessageEvent {isTrusted: true, data: {…}, origin: 'http://127.0.0.1:7860', lastEventId: '', source: Window, …}
index.js?a83964bb:3419 method Pause []
index.js?a83964bb:3419 return {target: Window, origin: 'http://127.0.0.1:7860', data: {…}}
index.js?a83964bb:3419 method Pause []
index.js?a83964bb:3419 return {target: Window, origin: 'http://127.0.0.1:7860', data: {…}}
index.js?a83964bb:3305 Infinity% loaded
index.js?1727603987.65641:1 Method Pause MessageEvent {isTrusted: true, data: {…}, origin: 'http://127.0.0.1:7860', lastEventId: '', source: Window, …}
index.js?1727603987.65641:1 Method Pause MessageEvent {isTrusted: true, data: {…}, origin: 'http://127.0.0.1:7860', lastEventId: '', source: Window, …}
index.js?a83964bb:3305 _o {isObject3D: true, uuid: 'c59f92df-0d4d-4d95-9ff4-031bf702228a', name: '', type: 'Group', parent: null, …}
index.js?a83964bb:3305 (2) [Wc, Wc]
index.js?a83964bb:3419 pushCommand
index.js?1727603987.65641:1 Uncaught (in promise) {method: 'Pause', status: 'Timeout'}

**Comment by NullEqualsZero** [comment]
My bad apparently it is built in ControlNet now
